config/min_balance.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2019-2025 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package config
+
+import (
+	"github.com/algorand/go-algorand/data/basics"
+)
+
+/* Functions that simplify the ways that ConsensusParams affect minimum balance
+   requirements. */
+
+// MinBalanceReq computes the minimum balance requirements for an account based on
+// some consensus parameters. MinBalance should correspond roughly to how much
+// storage the account is allowed to store on disk.
+func (proto *ConsensusParams) MinBalanceReq(u basics.AccountData) basics.MicroAlgos {
```

Comment:
it feels like `basics.AccountData.MinBalanceReq` is more natural than `config.ConsensusParams.MinBalanceReq`, idk